c2rust-analyze/src/main.rs

```diff
@@ -679,6 +698,35 @@ fn run(tcx: TyCtxt) {
         all_rewrites.extend(ty_rewrites);
     }
 
+    // Print results for `static` items.
+    eprintln!("\nfinal labeling for static items:");
+    for (did, lty) in gacx.static_tys.iter() {
+        let lcx1 = crate::labeled_ty::LabeledTyCtxt::new(tcx);
+        let name = tcx.item_name(*did);
+        let addr_of1 = gasn.perms[lty.label];
+        let ty1 = lcx1.relabel(lty, &mut |lty| {
+            if lty.label == PointerId::NONE {
+                PermissionSet::empty()
+            } else {
+                gasn.perms[lty.label]
+            }
+        });
+        eprintln!(
+            "{:?}: addr_of = {:?}, type = {:?}",
+            name,
+            //describe_local(tcx, decl),
+            addr_of1,
+            ty1,
+        );
```

Comment:
I'd also use inline formatting args (like `{name:?}`) here, as it's easy to mix up the order and filecheck needs them in a specific order (it happens with `perms` and `flags` in [fw/rewrite-never-mutated-static-muts-rebase](https://github.com/immunant/c2rust/tree/fw/rewrite-never-mutated-static-muts-rebase)).